pallets/multisig/src/lib.rs

```diff
@@ -203,6 +205,24 @@ decl_module! {
 
         fn deposit_event() = default;
 
+        fn on_runtime_upgrade() -> Weight {
+            use sp_version::RuntimeVersion;
+            use polymesh_primitives::migrate::kill_item;
+
+            // Kill pending proposals if the transaction version is upgraded
+            let current_version = <T::Version as Get<RuntimeVersion>>::get().transaction_version;
+            let last_version = TransactionVersion::get();
+            if last_version < current_version {
+                TransactionVersion::set(current_version);
+                for item in &["proposals", "proposal_detail", "proposal_ids", "Votes"] {
```

Comment:
```suggestion
                for item in &["Proposals", "ProposalIds", "ProposalDetail", "Votes"] {
```
We need to enter the variable name, not the function name here.